CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
+
+#### Breaking changes
+
+- In variables:
+  - Remove SimpleVariable and DatedVariable. Any Variable acts like old DatedVariable
```

Comment:
Ok for second line and the fanciful SimpleVariable.
In first line, what about AbstractVariable ? It was possible to create its own kind of variable before and we removed this service.